application/testing/CMakeLists.txt

```diff
@@ -866,7 +866,7 @@ f3d_test(NAME TestNonExistentTexture DATA cow.vtp ARGS --texture-material=${F3D_
 
 if(VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240707)
   # Test invalid geometry
-  f3d_test(NAME TestInvalidGeometry DATA duck_invalid.gltf ARGS --geometry-only REGEXP "failed to load geometry" NO_BASELINE)
+  f3d_test(NAME TestInvalidGeometry DATA world_invalid.obj ARGS --geometry-only REGEXP "failed to load geometry" NO_BASELINE)
```

Comment:
Why are you changing the test file here?

Reply:
because of a change in VTK (https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11130) that now lets us open the geometry of the duck_invalid file without issue. The duck_invalid can still not be opened as a full scene as expected.

I've created a new invalid file for this `--geometry-only` case